response_operations_ui/views/reporting_units.py

```diff
@@ -26,7 +27,17 @@ def view_reporting_unit(ru_ref):
             "title": f"{ru_ref}"
         }
     ]
-    return render_template('reporting-unit.html', ru=reporting_unit, breadcrumbs=breadcrumbs)
+
+    # Details for mock create message link
+    ru_details = {'survey': 'BRES 2017',
+                  'ru_ref': ru_ref,
+                  'business': 'Bolts & Rachets Ltd',
+                  'to': 'Jacky Turner',
```

Comment:
Should the respondent be hardcoded?

Reply:
For the moment yes, this is information we need to be passed from pages that don't exist yet.